src/components/ResultCard/ResultCard.tsx

```diff
@@ -44,20 +54,20 @@ export default function ResultCard(props: ResultCardOptions) {
         <p className='cio-result-card-title'>{result.value}</p>
         <div className='cio-result-card-details'>
           <div className='cio-result-card-rating'>
-            {ratingScore && (
+            {!!ratingScore && (
```

Comment:
Do any customer allow a `0` rating score? If so should we explicitly mention `ratingScore` to be defined

Reply:
Good callout. We discussed this and decided we want to hide these if the score is 0